fcli-other/fcli-functional-test/src/ftest/groovy/com/fortify/cli/ftest/ssc/SSCAlertDefinitionSpec.groovy

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2023 Open Text.
+ *
+ * The only warranties for products and services of Open Text 
+ * and its affiliates and licensors ("Open Text") are as may 
+ * be set forth in the express warranty statements accompanying 
+ * such products and services. Nothing herein should be construed 
+ * as constituting an additional warranty. Open Text shall not be 
+ * liable for technical or editorial errors or omissions contained 
+ * herein. The information contained herein is subject to change 
+ * without notice.
+ */
+package com.fortify.cli.ftest.ssc
+
+import static com.fortify.cli.ftest._common.spec.FcliSessionType.SSC
+
+import com.fortify.cli.ftest._common.Fcli
+import com.fortify.cli.ftest._common.spec.FcliBaseSpec
+import com.fortify.cli.ftest._common.spec.FcliSession
+import com.fortify.cli.ftest._common.spec.Prefix
+import com.fortify.cli.ftest.ssc._common.SSCAppVersion
+
+import spock.lang.AutoCleanup
+import spock.lang.Requires
+import spock.lang.Shared
+
+@Prefix("ssc.alert-definition") @FcliSession(SSC) 
+class SSCAlertDefinitionSpec extends FcliBaseSpec {
```

Comment:
This test seems to be assume that alert definitions exist, and even that a specific alert definition with id 84 exists; this may be true for the Fortify Hosted UAT environment, but this test will fail on my local SSC environment for example, as I don't have any alert definitions (at this moment). So, the `list` test should assume either `No Data` or any number of alert definitions. The `get` command should only be executed if the list command returned some data, just taking an arbitrary id from that list, but you'll need to investigate how to best implement something like that (or just disable the `get` test for now)